### DISCUSSION TITLE
how to create a live kernel dump via cli (using system informer or other tool)

### DISCUSSION THREAD

**Discussion by notsnakesilent** [winsiderss/systeminformer#1734] [Questions / Answers] [answered]
the title says all , thanks

**Comment by jxy-s** [comment, marked as answer]
There is no CLI in System Informer to achieve what you want. You will have to interact with the UI to do this with SI. But, LiveKD does have this capability: https://learn.microsoft.com/en-us/sysinternals/downloads/livekd

**Comment by dmex** [comment]
taskmgr.exe does have `-lkmd` command for live kernel dumps but it's not functional yet. 

You'll have to use the Powershell `Get-StorageDiagnosticInfo`  command:
https://www.elevenforum.com/t/create-live-kernel-dump-file-in-windows-11.8815/#Two
